### PR TITLE
Update Content Types page

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: #1040 

# What does this Pull Request do?

Minor updates to urls and image names. A touch of deCLAWing. 

# Interested parties
@seth-shaw-unlv

### CODE REVIEW

File: docs/user-documentation/content_types.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ RDF mapping aligns Drupal fields with RDF ontology properties. For example, the
...
 
-An RDF mapping configuration file has two main areas, the mapping's metadata and the mapping itself. Most of the mapping's metadata should be left alone unless you are creating a brand new mapping for a new Content Type or Taxonomy Vocabulary. A _partial_ example from [islandora_demo's islandora_object (Repository Item)](https://github.com/Islandora-CLAW/islandora_demo/blob/8.x-1.x/config/install/rdf.mapping.node.islandora_object.yml) is included below:
+An RDF mapping configuration file has two main areas, the mapping's metadata and the mapping itself. Most of the mapping's metadata should be left alone unless you are creating a brand new mapping for a new Content Type or Taxonomy Vocabulary. A _partial_ example from [islandora_demo's islandora_object (Repository Item)](https://github.com/Islandora/islandora_defaults/blob/8.x-1.x/config/install/rdf.mapping.node.islandora_object.yml) is included below:
```
Comment (whikloj) [inline docs/user-documentation/content_types.md]: The text of the link still says "islandora_demo's"

File: docs/user-documentation/content_types.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ The required mapping metadata fields when creating a brand-new mapping include t
...
 
-All the mappings use RDF namespaces instead of fully-qualified URIs. For example, the type for islandora_object is `pcdm:Object` instead of `http://pcdm.org/models#Object`. Unfortunately, the available namespaces are defined in module hooks (hook_rdf_namespaces) rather than in a configuration file. Repository managers wanting to add additional namespaces need to create their own module and implement hook_rdf_namespaces. See the [islandora_demo](https://github.com/Islandora-CLAW/islandora_demo/blob/8.x-1.x/islandora_demo.module) hook implementation for an example.
+All the mappings use RDF namespaces instead of fully-qualified URIs. For example, the type for islandora_object is `pcdm:Object` instead of `http://pcdm.org/models#Object`. Unfortunately, the available namespaces are defined in module hooks (hook_rdf_namespaces) rather than in a configuration file. Repository managers wanting to add additional namespaces need to create their own module and implement hook_rdf_namespaces. See the [islandora_defaults](https://github.com//Islandora/islandora_defaults/blob/8.x-1.x/islandora_demo.module) hook implementation for an example.
```
Comment (whikloj) [inline docs/user-documentation/content_types.md]: link text is changed but file also needs to change, should be `https://github.com//Islandora/islandora_defaults/blob/8.x-1.x/islandora_defaults.module`

Comment (whikloj) [reply docs/user-documentation/content_types.md]: oops, double slash

**Comment by manez** [comment]
@whikloj changes made

**Comment by whikloj** [comment]
Not part of your work but this looks like it was a placeholder that never got replaced https://github.com/Islandora/documentation/pull/1371/files#diff-f28077715874fdae17c1a431fd090633R50.
Probably needs to be `..., check out [Context](context.md)`

**Review by whikloj — APPROVED** [review]
👍

**Pull request merged**